cloudcix_primitives/bridgeifns.py

```diff
@@ -76,12 +76,12 @@ def build(
 
     status, config_data, msg = load_pod_config(config_file)
     if not status:
-      if config_data['raw'] is None:
-          return False, msg
-      else:
-          return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
-              indent=2,
-              sort_keys=True)
+        if config_data['raw'] is None:
+            return False, msg
+    else:
```

Comment:
This else statement is should be on the same level as the second if statement `if config_data['raw'] is None:`

https://github.com/CloudCIX/primitives/blob/b66525b84e0212ad8eb0e48fab2ed9093b8b9cc7/cloudcix_primitives/ns.py#L104